src/session.c

```diff
@@ -45,11 +45,11 @@ typedef struct SESSION_INSTANCE_TAG
     handle handle_max;
     uint32_t remote_incoming_window;
     uint32_t remote_outgoing_window;
-    int is_underlying_connection_open : 1;
+    uint8_t is_underlying_connection_open : 1;
```

Comment:
>uint8_t [](start = 4, length = 7)

Same comment (can we use unsigned int to be in sync with the others)? And seems that was fine on all compilers we ran up to now.

Reply:
same reply 😄

Reply:
unsigned int should be fine here too, I will leave it as is.

---
In reply to: [188157799](https://github.com/Azure/azure-uamqp-c/pull/227#discussion_r188157799) [](ancestors = 188157799)